libraries/lib-vst/VSTWrapper.h

```diff
@@ -329,5 +327,3 @@ struct VSTMessage : EffectInstance::Message
 };
 
 #endif
-
-#endif // USE_VST
```

Comment:
Is this complete?  `USE_VST` occurs in 15 files.

Reply:
I have force pushed the updated version.